tooling/cli/src/helpers/framework.rs

```diff
@@ -88,6 +96,8 @@ impl fmt::Display for Bundler {
 
 pub fn infer_from_package_json(package_json: &str) -> (Option<Framework>, Option<Bundler>) {
   let framework_map = [
+    ("solid-js", Framework::SolidJS, Some(Bundler::Vite)),
+    ("solid-start", Framework::SolidStart, Some(Bundler::Vite)),
```

Comment:
order matters here, `SolidStart` projects also includes a `solid-js` dependency. So we need to check for `solid-start` first.